Documentation/dependencies.md

```diff
@@ -32,6 +32,8 @@ For the most part the codebase is self-contained (e.g. all dependencies are vend
   * the stage1 binaries link against the library
 * libacl
   * development headers
+* libsystemd-journal
+  * development headers
```

Comment:
Since it is used by the API service, it is not only for building stage1 but for building rkt itself. Shouldn't it be moved to the previous section?

Same for the change in configure.ac